### PR TITLE
FOGL-2817: added package and Description for RPM packaging

### PR DESCRIPTION
**Pull request by pintomax**
FOGL-2817: added package and Description for RPM packaging

### CODE REVIEW

File: requirements.sh [hunk 1]
```diff
@@ -17,9 +17,25 @@
...
+foglamp_location=`pwd`
+os_name=`(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
+os_version=`(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
+echo "Platform is ${os_name}, Version: ${os_version}"
+
+if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *"7"* ]]; then
+	sudo yum -y install curl
+	sudo yum -y install libcurl
+	sudo yum -y install curl-devel
+	sudo yum -y install libcurl-devel
```
Comment (ashish-jabble) [inline requirements.sh:36]: remove `sudo` commands, the script needs to be executed with sudo

File: Package [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+plugin_name=slack
+plugin_type=notify
+plugin_install_dirname=slack
+
+# Now build up the runtime requirements list. This has 3 components
+#   1. Generic packages we depend on in all architectures and package managers
+#   2. Architecture specific packages we depend on
+#   3. Package manager specific packages we depend on
+requirements="foglamp,curl,libcurl,curl-devel,libcurl-devel"
```
Comment (ashish-jabble) [inline Package]: Add `foglamp-service-notification`, please match with `control` Debian files

File: requirements.sh (hunk 1 above)
Comment (pintomax) [reply requirements.sh:36]: If we remove sudo, and launch this script via sudo, then the permissions of created files are all set to root.

The only action needed by root is related to yum commands

File: Package (hunk 2 above)
Comment (pintomax) [reply Package]: yes, oversight

File: Description [hunk 3]
```diff
@@ -0,0 +1 @@
+C++ Slack notification delivery plugin package for FogLAMP.
```
Comment (MarkRiddoch) [inline Description:1]: I don't think we need to put the C++ bit in every description.

**Review by MarkRiddoch — APPROVED** [review]

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**